### ISSUE TITLE
Posts of people I don't follow appear on my home timeline

### ISSUE THREAD

**Issue by futureisfoss** [friendica/friendica#11762]
Look at this screenshot:
![image](https://user-images.githubusercontent.com/79040025/180482364-397a10b4-e7f9-4975-bd88-90b95547fe31.png)

Now I don't follow Anubis or Katherine, and I've never followed them actually, then why are their posts showing up on my home timeline?! And as you can see from my screenshot none of my followers have reshared or replied to the posts either, so this shouldn't be here right?
I've seen their posts before too, I believe this has been happening for months now, I just didn't pay close attention before because this wasn't my main account. And I'm also using the default settings of Friendica, see this screenshot:
![image](https://user-images.githubusercontent.com/79040025/180482619-ac54de32-4f89-4e52-9d89-76efb59e2bd9.png)

Any idea why this might be happening and how to fix this?

Tagging my instance admin @tio-trom here.

Note: I made a forum post about this 2 days ago - https://social.trom.tf/display/dbc8dc44-8762-d846-3c29-2af453092617
As you can see, a few people have responded saying they've had similar issues before.

**Comment by tio-trom** [comment]
Yes I can confirm I have seen several posts from the accounts on the same instance of ours, in my newsfeed. Some I have followed and then unfollowed and yet their posts came up in the newsfeed. This seems like a bad-bug :D. We run the latest Friendica version https://social.trom.tf/friendica

**Comment by annando** [comment]
Please hover over the symbol at the right of the username. Ir should tell why it is displayed.

**Comment by tio-trom** [comment]
> Please hover over the symbol at the right of the username. Ir should tell why it is displayed.

That's the first thing I checked and for me it only said "Public" and no more symbols other than that...that's why I was surprised.

**Comment by tio-trom** [comment]
I see an example now. This is a Friendica account mind you, from our instance:

https://user-images.githubusercontent.com/38043315/180605490-2d6e3392-f549-46aa-940e-581580903fd8.mp4

Basically I cannot get rid of the messages from this one, on my newsfeed....unless I block/ignore it.

**Comment by annando** [comment]
Please set the log level to debug. When a new post arrives, have a look for all entries in the log with the guid of the post. The guid is this combination of characters and numbers with the dashes behind the `/display/` of a post url.

**Comment by tio-trom** [comment]
Ok I will do this and hopefully send you something relevant today. Thanks!

**Comment by annando** [comment]
BTW: I just realized that I meant the icon right to the date - but this is obviously not existing for these posts.

**Comment by tio-trom** [comment]
Yes that's why it is weird...am still waiting for such a post to come up in the newsfeed so I can look at the logs...

**Comment by tio-trom** [comment]
The log file is 400MB and I can't really open and search through it....is too big. You want me to share it with you privately? I will try in the meantime to open it....

**Comment by tio-trom** [comment]
Ok it is reading it on my computer bit by bit. This is what I found:


```

2022-07-23T20:26:33Z worker [NOTICE]: Adding post for delayed publishing {"uid":76,"delayed":"2022-07-23 20:26:33","uri":"https://social.trom.tf/objects/dbc8dc44-d70476d6-1ee333a6d7f65e59","worker_id":"28b4356","worker_cmd":"OnePoll"} - {"file":"Delayed.php","line":81,"function":"add","uid":"7fd869","process_id":13454}

2022-07-23T20:26:43Z worker [NOTICE]: created item {"post-id":9132651,"uid":76,"network":"dfrn","uri-id":6640932,"guid":"dbc8dc44-d70476d6-1ee333a6d7f65e59","worker_id":"518e583","worker_cmd":"DelayedPublish"} - {"file":"Item.php","line":1145,"function":"insert","uid":"7fd869","process_id":13454}

[NOTICE]: Adding post for delayed publishing {"uid":76,"delayed":"2022-07-23 22:33:29","uri":"https://social.trom.tf/objects/dbc8dc44-60c5e835-a16155c302e775b8","worker_id":"4deee78","worker_cmd":"OnePoll"} - {"file":"Delayed.php","line":81,"function":"add","uid":"317b9b","process_id":24383}

[NOTICE]: created item {"post-id":9134796,"uid":76,"network":"dfrn","uri-id":6642013,"guid":"dbc8dc44-60c5e835-a16155c302e775b8","worker_id":"c9cde1b","worker_cmd":"DelayedPublish"} - {"file":"Item.php","line":1145,"function":"insert","uid":"317b9b","process_id":24383}

[DEBUG]: DELETE FROM `delayed-post`  WHERE (`uri` = 'https://social.trom.tf/objects/dbc8dc44-60c5e835-a16155c302e775b8' AND `uid` = 76) {"callstack":"Delayed::delete, Delayed::publish, DelayedPublish::execute, Worker::execFunction, Worker::execute, Worker::processQueue"} - {"file":"Database.php","line":1258,"function":"delete","uid":"317b9b","process_id":24383}
2022-07-23T22:33:40Z worker

[DEBUG]: twitter_hook_fork {"name":"notifier_normal","data":{"uid":76,"id":9134796,"parent":"9134796","uri-id":6642013,"uri":"https://social.trom.tf/objects/dbc8dc44-60c5e835-a16155c302e775b8","thr-parent":"https://social.trom.tf/objects/dbc8dc44-60c5e835-a16155c302e775b8","parent-uri":"https://social.trom.tf/objects/dbc8dc44-60c5e835-a16155c302e775b8","guid":"dbc8dc44-60c5e835-a16155c302e775b8","parent-guid":"dbc8dc44-60c5e835-a16155c302e775b8","received":"2022-07-23 22:33:39","created":"2022-07-23 22:08:15","edited":"2022-07-23 22:08:15","verb":"http://activitystrea.ms/schema/1.0/post","object-type":"http://activitystrea.ms/schema/1.0/note","object":"","target":"","private":0,"title":"","body":"[quote]Not according to Dr Ronald Pies. ‘A naughty antipsychiatrist did it and ran away.’ [url=https://www.psychiatrictimes.com/view/debunking-two-chemical-imbalance-myths-again]psychiatrictimes.com/view/debunking…[/url] [url=https://twitter.com/altostrata/status/1550917993713922048]twitter.com/altostrata/sta…[/url][/quote][hr][url]https://twitter.com/ClinpsychLucy/status/1550966045447430145[/url]","raw-body":"[quote]Not according to Dr Ronald Pies. ‘A naughty antipsychiatrist did it and ran away.’ [url=https://www.psychiatrictimes.com/view/debunking-two-chemical-imbalance-myths-again]psychiatrictimes.com/view/debunking…[/url] [url=https://twitter.com/altostrata/status/1550917993713922048]twitter.com/altostrata/sta…[/url][/quote][hr][url]https://twitter.com/ClinpsychLucy/status/1550966045447430145[/url]","location":"","coord":"","app":"Feed","inform":"","deleted":0,"extid":"","post-type":1,"post-reason":0,"gravity":0,"allow_cid":"","allow_gid":"","deny_cid":"","deny_gid":"","author-id":199989,"author-link":"https://social.trom.tf/profile/lucyjohnstone","author-name":"Lucy Johnstone (twitter mirror)","author-avatar":"https://social.trom.tf/photo/17519591926054b6a0d8b2a669354882-5.jpg?ts=1616164512","owner-id":199989,"owner-link":"https://social.trom.tf/profile/lucyjohnstone","contact-uid":76,"signed_text":null,"network":"dfrn","wall":1,"contact-id":199983,"plink":"https://twitter.com/ClinpsychLucy/status/1550966045447430145","origin":1,"thr-parent-id":6642013,"parent-uri-id":6642013,"postopts":"","pubmail":0,"event-created":null,"event-edited":null,"event-start":null,"event-finish":null,"event-summary":null,"event-desc":null,"event-location":null,"event-type":null,"event-nofinish":null,"event-ignore":null,"event-id":null},"execute":true} - {"file":"twitter.php","line":408,"function":"twitter_hook_fork","uid":"8ee705","process_id":29558}
2022-07-23T23:23:27Z worker
```
Am not sure what more to show you.

**Comment by tobiasd** [comment]
> The log file is 400MB and I can't really open and search through it....is too big. You want me to share it with you privately? I will try in the meantime to open it....

@annando might be able to give you some key words you can use to `grep` through the file. At one point a process ID is found, that you can `grep` for again to get all the log entries from that process.

If you have not done so, you should set up a log rotation job for the Friendica logs, to save your hard disk from filling up.

**Comment by futureisfoss** [comment]
I took a look at this icon you where talking about, the one beside the date. But after scrolling through my timeline a bit, I noticed this post that didn't contain that icon:
![image](https://user-images.githubusercontent.com/79040025/180655034-34f7da2c-63e7-4d9a-807b-35a37b846051.png)
This image might be showing up on my timeline because Tio replied to it and I follow Tio, but still it should be showing that icon there, right?

**Comment by tio-trom** [comment]
Yes for me it always shows the button when a friend interacted with a post...weird that it does not show for you...what if you change the theme to the default one?

**Comment by futureisfoss** [comment]
I should've been calling it "icon" instead of "button". I just edited my previous post and fixed that, haha 😅

No Tio, it shows me that icon when a friend interacted with a post, this is not a theme issue. See this post for example:
![image](https://user-images.githubusercontent.com/79040025/180655532-c46b9ecd-cb69-46d6-9859-96d88964a122.png)
I know this post is showing up because Blueberry replied to the post and I follow blueberry, and its showing me that icon too!
But the post I mentioned above where you replied to that person, for some reason it didn't show that icon, I don't know why. This is the link to that post if you need - https://social.trom.tf/display/f87d4b91-71afc6b8f3b3d41b-72f9b83f

**Comment by tio-trom** [comment]
Oh wow ok very weird...and probably related!?

**Comment by futureisfoss** [comment]
I just saw this post now:
![image](https://user-images.githubusercontent.com/79040025/180658251-9c08bc8b-82c7-4ad3-8438-153df07322a3.png)
Here is the link - https://social.trom.tf/display/dbc8dc44-4862-dd7a-99c1-75f640569983
@tio-trom can you check the log like @annando was saying?

**Comment by tio-trom** [comment]
I will try to but the log file is immense....

**Comment by annando** [comment]
For the next few days I will be busy with some other tasks, but I hope to have a look at this whole stuff some time later.

**Comment by tio-trom** [comment]
Another weird thing I just saw now. I see this post:
![2022-07-26_05-18](https://user-images.githubusercontent.com/38043315/180915672-e991a922-dcbc-48c0-9092-2f3da8dc15a6.png)

It is not from my own instance. However, someone on my instance that I do not follow shared the same post...:
![2022-07-26_05-18_1](https://user-images.githubusercontent.com/38043315/180915764-b96b8cfc-bc46-4cd4-bbef-bd828cb8030c.png)

So that's truly bizarre...almost as if this person from my instance sharing a post, triggers another account from Diaspora who shared the same link, to showcase in my personal newsfeed...I.Do.Not.Understand.Anymore. haha

**Comment by haheute** [comment]
I think I have the same problem.. I sometimes see posts from `@[askfedi_de](https://a.gup.pe/u/askfedi_de)` . It seems without somebody of my contacts liking it or commenting these posts. (although it says: "Stored because of a child post to complete this thread.")
strange.. I don't follow that group. (maybe one of my contacts follows that group...  I don't know)

**Comment by annando** [comment]
This post should have answers. Please have a look at all the answers and what the system there says about the post reason.

**Comment by haheute** [comment]
Ok. Yes, there are answers. They all show an `@` that says: "Du wurdest angeschrieben (bcc)." 
When opened in a private tab, it says: "Pushed to us" with another icon: 
https://libranet.de/display/52acf9cd-4098aeab26dbb4a9-65596bf5
https://libranet.de/display/17222a73-1f0043378a628f9b-75860092

**Comment by tio-trom** [comment]
I saw yet another one from a new Pixelfed account I do not follow:
![2022-08-15_12-47](https://user-images.githubusercontent.com/38043315/184622422-2a792f7b-fcb7-46f1-91e4-e37b1df3d88a.png)
It is almost like seeing ads :D. 

And I see no indication why this post shows in my Network Tab....is it possible to see it if anyone from my block/ignore list interacted with it? Maybe it is a Friendica bug?

**Comment by annando** [comment]
Which version are you on? is this a local or a remote contact?

**Comment by tio-trom** [comment]
`This is Friendica, version 2022.06 that is running at the web location https://social.trom.tf. The database version is 1469/1469, the post update version is 1452/1452.`

The account in question is not in my list of Friends. It is this post and account https://pixey.org/p/Bitwarden/464739366274552633 . The post I see it in my Network - should only see people I follow and such there.

**Comment by annando** [comment]
There had been a lot of changes in the current `develop`, so possibly this is already covered there. We will have to wait and see.
